api/WebGLRenderingContext.json

```diff
@@ -5657,6 +5657,101 @@
           }
         }
       },
+      "makeXRCompatible": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebGLRenderingContext/makeXRCompatibile",
+          "support": {
+            "chrome": {
+              "version_added": "79"
+            },
+            "chrome_android": {
+              "version_added": "79"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": false
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "WebGL2": {
```

Comment:
I think the WebGL2 entry here makes no sense, I would remove it altogether. 
If WebGL2 needs special listing, then the version numbers you've added here, make no sense as they are earlier than any XR implementation.

Reply:
I think it's a cut and paste error. Sorry.

Reply:
Fixed.